resource/zoneCreator/client.lua

```diff
@@ -63,7 +66,7 @@ local function closeCreator(cancel)
             }}
         }) or {}
 
-        format = input[2]
+        format = input[2] or 'array'
```

Comment:
There is no default format so if you submit the input without specifying one it errors on the server and you lose the zone. I added that because I figured it would be better to still get the data, even if it's the wrong format. Especially on a more complex polyzone, I'd rather have to reformat the code then remake the zone entirely

Reply:
Saving in the incorrect format is preferable to failing to save but I don't see how it's possible to encounter that issue in the first place. The `format` variable is always defined and and that is used as the default for that option.

Reply:
Ahh okay I see that now. I had it happen to me once I thought but it must have just been a one off or maybe something else entirely. I've removed the line

Reply:
Still need the line, just reverse the changes

Reply:
Right sorry, all over the place today. Fixed now